pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -448,8 +448,6 @@ func waitLogs(ctx context.Context, id string, w *fsnotify.Watcher, runtimeServic
 			case fsnotify.Rename:
 				fallthrough
 			case fsnotify.Remove:
-				fallthrough
-			case fsnotify.Chmod:
 				return true, true, nil
 			default:
 				klog.ErrorS(nil, "Received unexpected fsnotify event, retrying", "event", e)
```

Comment:
If `case fsnotify.Chmod:` is deleted, fsnotify.Chmod event is judged as `default:` and logged as an unexpected fsnotify event error.
It seems that we need to consider whether it is appropriate.
Although fsnotify.Chmod event may occur at an unexpected times according to your report, not an unexpected kind of event.
And, fsnotify.Chmod will be indistinguishable from a truly unexpected event.

How about not deleting `case fsnotify.Chmod:` and not processing anything?
However, I think we need to check that not handling this event will not affect existing processing on any architecture.
Because there might be a reason why it has been handled this event so far.

Reply:
@cyclinder can you check this comment?

Reply:
/cc @giuseppe 
from earlier change

It seems to me that we're not actually using whether the log file was found anywhere: https://github.com/kubernetes/kubernetes/blob/fbffe056dd03bd9c746e8819ad22043d640a4489/pkg/kubelet/kuberuntime/logs/logs.go#L429-L432

I would expect chmod would neither create a new file nor recreate the existing one, so I'd expect a return value of something like `false, false, nil` for this case. 

I agree that Chmod is an expected case and we should handle it.

Are there no accompanying test changes to ensure this fix works?

Reply:
I agree with @ehashman, the chmod event should be handled.

I don't see why it should behave differently on arm, it could be a kernel bug, or an issue in `github.com/fsnotify/fsnotify`.

Do you have a reproducer with fsnotify that returns the chmod event (which is mapped to the kernel event `IN_ATTRIB`) when there are no changes happening to the file?

And if it behaves differently on arm, could you strace it and see what events are received from the kernel?